ksqldb-api-client/src/main/java/io/confluent/ksql/api/client/impl/AdminResponseHandlers.java

```diff
@@ -335,7 +335,9 @@ static boolean isConnectErrorResponse(final JsonObject ksqlEntity) {
           source.getString("valueFormat"),
           formatQueries(source.getJsonArray("readQueries")),
           formatQueries(source.getJsonArray("writeQueries")),
-          Optional.ofNullable(source.getString("timestamp")),
+          source.getString("timestamp").isEmpty()
```

Comment:
Can't getString("timesetamp" be null?

Reply:
Not sure... I just re-established the old code... \cc @vcrfxia do you know?

Reply:
No, it cannot be null: https://github.com/confluentinc/ksql/blob/05f5f617dbc30296c048ae0a421a1a9d97b5fead/ksqldb-rest-app/src/main/java/io/confluent/ksql/rest/entity/SourceDescriptionFactory.java#L94

Reply:
Ideally this code would've been more defensive but the real fix we want is https://github.com/confluentinc/ksql/issues/6042 anyway.